packages/gravity-ui/.gitignore

```diff
@@ -0,0 +1,122 @@
+# Exclude build Output
```

Comment:
Would it perhaps be better to use either the full `gravity-ui-web` name for this directory? Feels more consistent.

Also, it's conceivable that other `gravity-ui-*` packages may one day live in this monorepo. `gravity-ui-nunjucks` and/or `gravity-ui-react` perhaps?

Reply:
Changed to `gravity-ui-web`